packages/block-library/src/query-title/edit.js

```diff
@@ -174,6 +177,60 @@ export default function QueryTitleEdit( {
 		);
 	}
 
+	if ( type === 'post-type-label' ) {
+		let title;
+		if ( postTypeLabel ) {
+			if ( showPrefix ) {
+				title = sprintf(
+					/* translators: %s: Singular post type name of the queried object */
+					__( 'Post Type: "%s"' ),
+					postTypeLabel
+				);
+			} else {
+				title = postTypeLabel;
+			}
+		} else {
+			title = showPrefix ? __( 'Post Type: Name' ) : __( 'Name' );
+		}
+
+		titleElement = (
+			<>
+				<InspectorControls>
+					<ToolsPanel
+						label={ __( 'Settings' ) }
+						resetAll={ () =>
+							setAttributes( {
+								showPrefix: true,
+							} )
+						}
+						dropdownMenuProps={ dropdownMenuProps }
+					>
+						<ToolsPanelItem
+							hasValue={ () => ! showPrefix }
+							label={ __( 'Show post type label' ) }
+							onDeselect={ () =>
+								setAttributes( { showPrefix: true } )
+							}
+							isShownByDefault
+						>
+							<ToggleControl
+								__nextHasNoMarginBottom
+								label={ __( 'Show post type label prefix' ) }
```

Comment:
```suggestion
								label={ __( 'Show post type label' ) }
```

<img width="553" height="231" alt="Image" src="https://github.com/user-attachments/assets/2495a292-341f-4c2e-9875-2a9ca73fd7f8" />

It's generally a good idea to match the labels for the dropdown items and toggles. Personally, I prefer simple labels.